eslint-rules/no-add-event-listener.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+    meta: {
+        type: 'suggestion',
+        docs: {
+            description: 'Enforce usage of addEventListener from @utils instead of native addEventListener',
+            category: 'Best Practices',
+            recommended: true,
+        },
+        fixable: 'code',
+        schema: [], // no options
+    },
+
+    create(context) {
+        return {
+            CallExpression(node) {
+                // Check if it's an addEventListener call
+                const callee = node.callee
+                if (callee.type === 'MemberExpression' && callee.property.name === 'addEventListener') {
+                    context.report({
+                        node,
+                        message: 'Use addEventListener from @utils instead of calling it directly on elements',
+                        fix(fixer) {
+                            // Get the element expression
+                            const elementText = context.getSourceCode().getText(callee.object)
+
+                            // Get the event type
+                            const eventText = context.getSourceCode().getText(node.arguments[0])
+
+                            // Get the callback
+                            const callbackText = context.getSourceCode().getText(node.arguments[1])
+
+                            // Get options if they exist
+                            const optionsText =
+                                node.arguments[2] != null
+                                    ? context.getSourceCode().getText(node.arguments[2]) === 'true'
+                                        ? ', { capture: true }'
+                                        : `, ${context.getSourceCode().getText(node.arguments[2])}`
+                                    : ''
+
+                            // Add import if needed (note: this is a basic implementation, it won't always work)
+                            const importFix = fixer.insertTextBefore(
+                                context.getSourceCode().ast,
+                                "import { addEventListener } from './utils'\n"
+                            )
```

Comment:
Wondering if this makes any sense. Most often than not it'll be correct, but it will be a broken import sometimes

Maybe we shouldn't be autofixing this at all?

Reply:
yeah, I'd leave the fix (if we can't detect the correct path). most folks' editors should be able to create the import on demand anyway

Reply:
I'll see if `eslint` allows me to detect what file I'm in, if so, I'll keep it